SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialClassInfoManager.cpp

```diff
@@ -507,3 +507,46 @@ void USpatialClassInfoManager::QuitGame()
 	FGenericPlatformMisc::RequestExit(false);
 #endif
 }
+
+uint32_t USpatialClassInfoManager::ComputeActorInterestComponentId(AActor* Actor)
```

Comment:
If this function is returning a component Id, can we use the custom type we have for it?

Reply:
If we like this change there might be other places we can change uint32_t for Worker_ComponentId.

Reply:
Const AActor*?